### ISSUE TITLE
Shift+alphabet don't work when disconnected

### ISSUE THREAD

**Issue by tamo** [ueno/ibus-fep#15]
Seems like a side-effect of the fix for #14

Env: ibus-1.4.1 and ibus-skk

Steps:
1. Ctrl+SPC to disable ibus (Status: [あ] -> [  ])
2. Press Shift+A

Expected: "A"

Actual result: nothing

``` plaintext
1335252206.356956 DEBUG write KEY_EVENT 42000000 01000000 42     # Shift + B
1335252206.419304 DEBUG read RESPONSE 06 01000000
1335252206.437138 DEBUG read SEND_TEXT 4200
1335252206.455784 DEBUG pty read "B"                             # OK
1335252209.891789 DEBUG write KEY_EVENT 20000000 04000000 00     # disable ibus
1335252210.113363 DEBUG read RESPONSE 06 01000000
1335252210.157630 DEBUG read SET_STATUS_TEXT 5B20205D2000 03000000000000000000000000000000
1335252211.438546 DEBUG write KEY_EVENT 42000000 01000000 42     # the same key, a different result
1335252211.445324 DEBUG read RESPONSE 06 01000000
1335252211.457693 DEBUG read FORWARD_KEY_EVENT 42000000 01000000
1335252211.457878 WARNING no tty sequence for key 66 modifiers 1
1335252215.203420 DEBUG write KEY_EVENT 41000000 01000000 41
1335252215.210149 DEBUG read RESPONSE 06 01000000
1335252215.222426 DEBUG read FORWARD_KEY_EVENT 41000000 01000000
1335252215.222545 WARNING no tty sequence for key 65 modifiers 1
1335252216.770711 DEBUG write KEY_EVENT 43000000 01000000 43
1335252216.777500 DEBUG read RESPONSE 06 01000000
1335252216.789812 DEBUG read FORWARD_KEY_EVENT 43000000 01000000
1335252216.789931 WARNING no tty sequence for key 67 modifiers 1
1335252218.55628 DEBUG write KEY_EVENT 44000000 01000000 44
1335252218.62412 DEBUG read RESPONSE 06 01000000
1335252218.74722 DEBUG read FORWARD_KEY_EVENT 44000000 01000000
1335252218.74836 WARNING no tty sequence for key 68 modifiers 1
1335252220.257060 DEBUG write KEY_EVENT 45000000 01000000 45
1335252220.263875 DEBUG read RESPONSE 06 01000000
1335252220.276116 DEBUG read FORWARD_KEY_EVENT 45000000 01000000
1335252220.276235 WARNING no tty sequence for key 69 modifiers 1
1335252220.686713 DEBUG write KEY_EVENT 46000000 01000000 46
1335252220.693505 DEBUG read RESPONSE 06 01000000
1335252220.705782 DEBUG read FORWARD_KEY_EVENT 46000000 01000000
1335252220.705900 WARNING no tty sequence for key 70 modifiers 1
```

**Comment by ueno** [comment]
Thanks.  It should be fixed with ueno/libfep@50207bfc12f945e5dbb2b9f181d99200070f945f.

**Comment by tamo** [comment]
Confirmed!
